certified-connectors/YakChat/Readme.md

```diff
@@ -0,0 +1,14 @@
+
+## YakChat Connector
+The YakChat connector allows you to send and receive SMS and MMS using your YakChat numbers. YakChat adds SMS and MMS to Microsoft Teams so that you can connect instantly with your customers and co-workers using text messaging.
+

```

Comment:
@aMatthewsYakChat Please refer the [templates provided for readme.md file]
(https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) and add required sections.,

Reply:
This has been updated